package.json

```diff
@@ -46,6 +46,6 @@
     "sinon": "^9.0.2"
   },
   "engines": {
-    "node": ">=12.13.0"
+    "node": ">=12.13.0 <13 || >=14"
```

Comment:
I don't mind not having this addition since Node 13 is already EOL and users shouldn't use it.